pkg/services/alerting/notifiers/victorops.go

```diff
@@ -22,22 +22,22 @@ const alertStateRecovery = "RECOVERY"
 func init() {
 	alerting.RegisterNotifier(&alerting.NotifierPlugin{
 		Type:        "victorops",
-		Name:        "VictorOps",
-		Description: "Sends notifications to VictorOps",
-		Heading:     "VictorOps settings",
+		Name:        "Splunk OnCall",
+		Description: "Sends notifications to Splunk OnCall",
+		Heading:     "Splunk OnCall settings",
 		Factory:     NewVictoropsNotifier,
```

Comment:
We should also rename the code https://github.com/grafana/alerting/tree/main/receivers/victorops and the functions that are used here.